db-es-audit/src/find_granule_counts.py

```
@@ -0,0 +1,616 @@
+                        "size":0,
+                        "track_total_hits": True}
+
+    elastic_response = requests.get(f"{GRAN_ELASTIC_URL}/{index}/_search",
+                                                data=json.dumps(elastic_base_query),
+                                                headers=headers)
+    if elastic_response.status_code >= 300:
+        logger.error(f"ERROR: request to elastic failed with error: {elastic_response.text}")
+        sys.exit(1)
+
+    granule_hits = elastic_response.json()["hits"]["total"]["value"]
+    logger.debug("granule hits\n", granule_hits)
```

Ruff (0.15.12)

[error] Too many arguments for `logging` format string

(PLE1205)